apollo-router/src/plugins/telemetry/config_new/instruments.rs

```diff
@@ -985,6 +987,54 @@ impl InstrumentsConfig {
                     .init(),
             ),
         );
+
+        // This new metric will also have the pipeline information.
+        // The reason that this instrument has a different name to the one above is that we wish to also implement this in router 2.0
```

Comment:
This entire comment can probably go on 2.x since the other metric doesn't exist here.
```suggestion
        // This new metric will also have the pipeline information.
        // The reason that this instrument has a different name to the one above is that we wish to also implement this in router 2.0
```

Reply:
[a99b852](https://github.com/apollographql/router/pull/7023/commits/a99b8522422458273297bf098ff2e62c9108d049)